test/todo_explanation.js

```diff
@@ -0,0 +1,58 @@
+var tap = require('tap');
+var tape = require('../');
+var concat = require('concat-stream');
+
+var common = require('./common');
+var stripFullStack = common.stripFullStack;
+
+tap.test('tape todo test', function (assert) {
+    var test = tape.createHarness({ exit: false });
+    assert.plan(1);
+
+    test.createStream().pipe(concat(function (body) {
+        assert.equal(
+            stripFullStack(body.toString('utf8')),
+            'TAP version 13\n'
+            + '# success\n'
+            + 'ok 1 this test runs\n'
+            + '# TODO incomplete test1\n'
+            + 'not ok 2 needs insight # TODO\n'
+            + '  ---\n'
+            + '    operator: fail\n'
+            + '    at: Test.<anonymous> ($TEST/todo_explanation.js:$LINE:$COL)\n'
```

Comment:
the test on 0.8 seems to want `Test.tap.test.test.todo` here instead of `Test.<anonymous>` :-/

I'm confused because we have many other tests using `Test.<anonymous>` that don't have special handling for node 0.8.

Reply:
I get that, but only why node 0.8 😭 I've gone through the source and debugged it a thousand times but only node 0.8 failes for only `todo_explanation.js`. It's just a copy of `todo.js`

Reply:
I'm wondering if the change to `emit('prerun') somehow changed things for the v8 version 0.8 is using.

We could update `test.common`, perhaps, for node 0.8 only, to swap out `Test.tap.test.test.todo` for `Test.<anonymous>`?

Reply:
> I'm wondering if the change to `emit('prerun') somehow changed things for the v8 version 0.8 is using.

I tried by using the latest commit from `master`, It still has the same problem.

My [naive] understanding is that in V8 version of 0.8, the `stack` part assumes the `{todo: 'something'}` object as one of the stack layer. The same happens for my local `skip_explanation.js` test. The test in https://github.com/substack/tape/pull/473#discussion_r296560029 comment has the function in stack pointing to `Test.tap.test.platform`.